src/Engine/RenderBackend/Text/TextRenderer.cpp

```
@@ -0,0 +1,116 @@
+    Font2MeshMap::const_iterator it = font2MeshMap.find(fontKey);
+    if (it != font2MeshMap.end()) {
+        textMesh = it->second;
+        if (textMesh == nullptr) {
+            osre_debug(Tag, "Invalid font mesh detected.");
+            return;
+        } else {
+            getMeshInfo(textMesh, meshInfoArray, info);
+        }
+    }
+
+    const ui32 lastIndex = textMesh->getLastIndex();
```

cppcheck

[warning] Possible null pointer dereference

(nullPointer)